babel.config.js

```diff
@@ -2,6 +2,7 @@
 // See LICENSE.txt for license information.
 
 const config = {
+    sourceType: 'unambiguous',
```

Comment:
Do you remember what this was needed for? It sounds like something that'd make sense to have, but I'm curious if there was something specific that needed it

Reply:
Looks like we actually don't need it. Removed.